app/controllers/redirect/help_center_controller.rb

```diff
@@ -9,17 +9,16 @@ def show
     private
 
     def validate_help_center_article_params
-      begin
-        return if MarketingSite.valid_help_center_article?(**article_params)
-      rescue ActionController::ParameterMissing
-      end
-
+      return if MarketingSite.valid_help_center_article?(**article_params)
       redirect_to root_url
     end
 
     def article_params
-      category, article = params.require([:category, :article])
-      { category: category, article: article }
+      begin
+        category, article = params.require([:category, :article])
+      rescue ActionController::ParameterMissing; end
```

Comment:
What's the purpose of the `.require` and the `rescue` here? That it's both or none?

I feel like we could do `values_at` which is happy enough if one or both values are `nil`?

```suggestion
      category, article = params.values_at(:category, :article)
```

Reply:
Updated in 6cf3f36